packages/components/src/components/Tabs.vue

```diff
@@ -92,15 +92,15 @@ export default {
   background-color: $black;
 
   &__header {
-    position: absolute;
```

Comment:
@ty we can't change this property because it fixes a bug. See [here](https://github.com/getappmap/appmap-js/issues/1015).